libs/checkpoint/langgraph/store/base/__init__.py

```diff
@@ -120,6 +183,12 @@ class PutOp(NamedTuple):
       - Values can be of any serializable type
     - If None, it indicates that the item should be deleted
     """
+    index: Optional[bool] = None  # type: ignore[assignment]
+    """Whether to index the item (if supported by the store).
+    
+    Defaults to True if the store supports indexing. This will embed the document
+    so it can be queried using search.
```

Comment:
technically can still find. Clarify it would be scored when providing a query, etc.